abcclassroom/github.py

```diff
@@ -182,6 +307,27 @@ def init_and_commit(directory, custom_message=False):
         print("No changes to local repository.")
 
 
+def _master_branch_to_main(directory):
+    """Change the name of the master branch to main
+
+    Changes the name of the master branch to main for the repo in the
+    given directory. Since we create the repo on github first, which now sets
+    the default branch to 'main', we need the local repo to match
+    in order to be able to push with error later.
+    """
+    print(
```

Comment:
SO this runs each time even if the main branch exists. i think the thinking probably was for `new_template` you'd only run it once but i use it all of the time because the update template opens an annoying text editor. i think this would be more robust if it checked to see if main exists and renames accordingly. i could see this being problematic however if catching git returns is hard. 

This works but how hard is it to capture git output at the CLI using subprocess calls?
```
(base) CIRES-EL-LM-020:my-assignment-template leahwasser$ git rev-parse --verify main
070a6cc0cca92cb3dc034fe1832529eb54fc7afb
(base) CIRES-EL-LM-020:my-assignment-template leahwasser$ git rev-parse --verify mainwe
fatal: Needed a single revision
```